### PR TITLE
Add isDisabled, selectProps and showTotalInfo props.

### PR DESCRIPTION
**Pull request by ISp1rit**

### CODE REVIEW

File: packages/pagination/src/Pagination.tsx [hunk 1]
```diff
@@ -20,9 +20,12 @@ const PurePagination: React.FC<IPagination> = ({
   currentPage = 1,
   visibleRange = 5,
   showSizeChanger = true,
+  isDisabled = false,
```
Comment (tundraray) [inline packages/pagination/src/Pagination.tsx:23]: plz rename this props like in other components. just disabled will be correct

File: packages/pagination/src/interfaces.ts [hunk 2]
```diff
@@ -1,25 +1,29 @@
...
 export interface IPagination {
   currentPage?: number;
   showSizeChanger?: boolean;
+  isDisabled?: boolean;
+  showTotalInfo?: boolean;
   pageSize?: number;
   pageSizeOptions?: number[];
   visibleRange?: 3 | 5 | 7;
   total: number;
+  selectProps?: SelectProps;
```
Comment (tundraray) [inline packages/pagination/src/interfaces.ts:18]: What is it selectProps? I do not understand what this parameter means?

File: packages/pagination/src/interfaces.ts [hunk 3]
```diff
@@ -1,25 +1,29 @@
...
+import { ISelectBaseTheme, SelectProps } from '@xcritical/select';
 
 import { ButtonGroupTheme, ButtonTheme } from '@xcritical/button';
 
 
 export interface IPagination {
   currentPage?: number;
   showSizeChanger?: boolean;
+  isDisabled?: boolean;
+  showTotalInfo?: boolean;
```
Comment (tundraray) [inline packages/pagination/src/interfaces.ts:13]: showTotals - it will be shorter

File: packages/pagination/src/interfaces.ts (hunk 2 above)
Comment (ISp1rit) [reply packages/pagination/src/interfaces.ts:18]: If I use showSizeChanger it renders select. I need it to be configurable.
You want me to rename it to sizeChangerProps?

**Pull request merged**